Packs/Zendesk/Integrations/Zendeskv2/README.md

```diff
@@ -480,6 +481,7 @@ Required permissions: Agents.
 | collaborators | Users to add as CCs when creating a ticket. | Optional | 
 | comment | A comment to add to the ticket. | Optional | 
 | html_comment | An HTML comment to add to the ticket. | Optional | 
+| public | true if a public comment; false if an internal note. The initial value set on ticket creation persists for any additional comment unless you change it. Possible values are: true, false. Default is true. | Optional | 
```

Comment:
comment or note? what's the right term?

Reply:
This is from the api docs - comment will be the right term